rs/ic_os/config/src/update_config.rs

```diff
@@ -0,0 +1,518 @@
+use crate::serialize_and_write_config;
+use anyhow::{anyhow, Context, Result};
+use macaddr::MacAddr6;
+use std::collections::HashMap;
+use std::ffi::OsStr;
+use std::fs;
+use std::net::{Ipv4Addr, Ipv6Addr};
+use std::path::Path;
+use url::Url;
+
+use config_types::*;
+
+pub static CONFIG_ROOT: &str = "/boot/config";
+pub static STATE_ROOT: &str = "/var/lib/ic/data";
+
+pub fn update_guestos_config() -> Result<()> {
+    let config_dir = Path::new(CONFIG_ROOT);
+    log_directory_structure(config_dir)?;
+    let state_root = Path::new(STATE_ROOT);
+    log_directory_structure(state_root)?;
```

Comment:
I can't remember, what is in `STATE_ROOT` that we want to log?

Reply:
All this good stuff: https://github.com/dfinity/ic/blob/master/ic-os/components/init/bootstrap-ic-node/bootstrap-ic-node.sh#L84